spreadsheets.qmd

```diff
@@ -630,10 +628,10 @@ write_sheet(bake_sale, ss = "bake-sale", sheet = "Sales")
 
 ### Authentication
 
-While you can read from a public Google Sheet without authenticating with your Google account, reading a private sheet or writing to a sheet requires authentication so that googlesheets4 can view and manage *your* Google Sheets.
+While you can read from a public Google Sheet with `gs4_deauth()`: without authenticating with your Google account, reading a private sheet or writing to a sheet requires authentication so that googlesheets4 can view and manage *your* Google Sheets.
```

Comment:
```suggestion
While you can read from a public Google Sheet without authenticating with your Google account and with `gs4_deauth()`, reading a private sheet or writing to a sheet requires authentication so that googlesheets4 can view and manage *your* Google Sheets.
```